Neuro/Minigames/Completion/Solvers/OxygenDepletedSolver.cs

```diff
@@ -0,0 +1,29 @@
+﻿
+
+using Neuro.Cursor;
+using System;
+using System.Collections;
+
+namespace Neuro.Minigames.Completion.Solvers;
+
+[MinigameSolver(typeof(KeypadGame))]
+public sealed class OxygenDepletedSolver : MinigameSolver<KeypadGame>
```

Comment:
This doesn't work.

1. On skeld, solving either of the two O2's will mark the one in O2 as complete, even if the one in Admin was solved.
2. When opening the second sabotage (in Admin), the keypad will not work.

Reply:
Darn. My fault for only testing it on only one of the keypads. I will bring this one back to the drawing board.

Reply:
Fixed in [7c8e262](https://github.com/VedalAI/neuro-amongus/pull/30/commits/7c8e26284f3b2382aa39116c353e39d9b0ab3a4d)